package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@revenuecat/purchases-js",
-  "version": "0.18.2",
```

Comment:
I believe this change is just a leftover from the merge and release of the new version this Monday. It happened in my branch too earlier today so we can just keep it.